package.json

```diff
@@ -33,12 +34,12 @@
     "babylon": "^6.18.0",
     "cli-color": "^1.2.0",
     "difflib": "^0.2.4",
-    "eslint": "^4.19.1",
     "fluent": "^0.6.4",
     "fluent-syntax": "^0.7.0",
     "gh-pages": "^1.1.0",
     "gitbook-cli": "^2.3.0",
-    "json-diff": "^0.5.2"
+    "json-diff": "^0.5.2",
+    "prettier": "^1.17.1"
```

Comment:
This isn't recommended practice by prettier, https://prettier.io/docs/en/install.html.

Reply:
Thanks. This sounds like a good practice, indeed.